### PR TITLE
Inventory-like GUI for stripping.

### PR DESCRIPTION
**Pull request by sentient-ai**
Apparently calls to ItemSlotButton.OnPressed with left/right clicks send 2 calls back to back. Middle's don't.
Double calls break shit with null-shenaniganry when you try to shove nonexistent things on peoples bodies.

Have not figured out hands/cuffs, or even the sprites to fill the GUI. Plagiarism only gets me so far.

All negative input on this monstrosity appreciated.

### CODE REVIEW

**Comment by sentient-ai** [comment]
https://streamable.com/w4hxgp

okay. fixed the weird double-call interaction, hands work, sprites work.
i'm pretty happy with it now.
just accepting that uncuffing oughta be done from the right click menu, unless there's more reworks to be had.

File: Content.Shared/GameObjects/Components/Inventory/EquipmentSlotDefinitions.cs [hunk 1]
```diff
@@ -47,6 +47,8 @@ public enum Slots
             POCKET4,
             EXOSUITSLOT1,
             EXOSUITSLOT2,
+            LHAND,
+            RHAND,
```
Comment (Zumorica) [inline Content.Shared/GameObjects/Components/Inventory/EquipmentSlotDefinitions.cs:51]: This is WAY too hardcoded... Currently you can add hands to any humans with a simple command, so it'd be best if you didn't hardcode slots like this, and instead handled multiple hands sanely.

**Review by Zumorica — CHANGES_REQUESTED** [review]
Read my review. I'd personally recommend you handle hands separately from other slots.

**Comment by Zumorica** [comment]
@sentient-ai is it fine if I take over this and finish it myself?

**Comment by SweptWasTaken** [comment]
Please finish this PR.

**Comment by sentient-ai** [comment]
I completely missed the comment by Zum two months ago. Really got stuck on finding a way to get the extra hands / handcuffs working and dropped off the face of the earth, sorry.

**Comment by Zumorica** [comment]
I can take it off your hands if you want me to, your call

**Comment by sentient-ai** [comment]
Please do, don't feel qualified to do it justice at the moment. If I keep contributing I'll stick to lower level stuff.

**Comment by SweptWasTaken** [comment]
> 
> 
> I can take it off your hands if you want me to, your call

15 days and no pulse, closing.

**Pull request closed without merging**